front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Parent component for both UserMessage and AgentMessage, to ensure avatar,
+ * side buttons and spacing are consistent between the two
+ */
+export function ConversationMessage({
+  avatar,
+  name,
+  children,
+}: {
+  avatar: React.ReactNode | null;
+  name: string | null;
+  children?: React.ReactNode;
+}) {
+  return (
+    <>
+      <div className="flex-shrink-0">{avatar || ""}</div>
```

Comment:
I supposed empty string vs null is about the asme but I find it clearer to return null when it's null.

Reply:
:+1: I was afraid that the null would be rendered but since it's not, fully agree